### ISSUE TITLE
Clicking on test running in ios simulator does not show finders

### ISSUE THREAD

**Issue by eseidelGoogle** [flutter/flutter#4097]
`flutter run -t test/my_test.dart`

See it run in your ios simulator.  Click on it.  No logs?

Maybe I'm holding it wrong.  @Hixie

**Comment by Hixie2** [comment]
If you stick a breakpoint in event handling code, does it break when you tap?

**Comment by TahaTesser** [comment]
This seems to be working fine  using default widget_test.dart on iPhone 11 simulator

<details> 
<summary>logs</summary> 

``` 
taha@Tahas-MacBook-Pro dev_flutter % flutterd run -t test/widget_test.dart -d iphone
Launching test/widget_test.dart on iPhone 11 in debug mode...
 
Running Xcode build...                                                  
                                                   
 └─Compiling, linking and signing...                        21.7s
Xcode build done.                                           38.1s
Waiting for iPhone 11 to report its views...                         3ms
flutter: 00:00 +0: Counter increments smoke test                        
Syncing files to device iPhone 11...                               284ms

Flutter run key commands.
r Hot reload. 🔥🔥🔥
R Hot restart.
h Repeat this help message.
d Detach (terminate "flutter run" but leave application running).
c Clear the screen
q Quit (terminate the application on the device).
An Observatory debugger and profiler on iPhone 11 is available at: http://127.0.0.1:64781/qsMS8DUjLi8=/
flutter: 00:00 +1: All tests passed!
flutter: Some possible finders for the widgets at Offset(451.2, 302.9):
flutter:   find.text('Test finished.')
flutter:   find.byType(Center)
flutter:   find.byType(Container)
flutter: Some possible finders for the widgets at Offset(449.3, 289.4):
flutter:   find.text('Test finished.')
flutter:   find.byType(Center)
flutter:   find.byType(Container)
flutter: No widgets found at Offset(320.8, 256.5).
flutter: No widgets found at Offset(203.9, 320.3).
flutter: No widgets found at Offset(530.4, 350.2).
flutter: No widgets found at Offset(452.2, 338.6).
flutter: No widgets found at Offset(287.0, 252.7).
flutter: No widgets found at Offset(372.9, 251.7).
flutter: Some possible finders for the widgets at Offset(354.6, 320.3):
flutter:   find.text('Test finished.')
flutter:   find.byType(Center)
flutter:   find.byType(Container)
flutter: Some possible finders for the widgets at Offset(355.6, 313.5):
flutter:   find.text('Test finished.')
flutter:   find.byType(Center)
flutter:   find.byType(Container)
flutter: No widgets found at Offset(436.7, 326.1).
flutter: Some possible finders for the widgets at Offset(377.8, 302.9):
flutter:   find.text('Test finished.')
flutter:   find.byType(Center)
flutter:   find.byType(Container)
flutter: Some possible finders for the widgets at Offset(419.3, 312.6):
flutter:   find.text('Test finished.')
flutter:   find.byType(Center)
flutter:   find.byType(Container)
flutter: Some possible finders for the widgets at Offset(465.7, 321.3):
flutter:   find.text('Test finished.')
flutter:   find.byType(Center)
flutter:   find.byType(Container)
flutter: No widgets found at Offset(487.9, 326.1).
flutter: No widgets found at Offset(513.0, 328.0).

``` 

</details> 


<details> 
<summary>flutter doctor -v</summary> 

```
[✓] Flutter (Channel dev, 1.21.0-1.0.pre, on Mac OS X 10.15.6 19G73, locale en-GB)
    • Flutter version 1.21.0-1.0.pre at /Users/taha/Code/flutter_dev
    • Framework revision f25bd9c55c (6 days ago), 2020-07-14 20:26:01 -0400
    • Engine revision 99c2b3a245
    • Dart version 2.9.0 (build 2.9.0-21.0.dev 20bf2fcf56)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/taha/Code/sdk
    • Platform android-29, build-tools 29.0.3
    • ANDROID_HOME = /Users/taha/Code/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.6)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.6, Build version 11E708
    • CocoaPods version 1.9.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 47.1.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.47.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.12.2

[✓] Connected device (5 available)
    • SM M305F (mobile)      • 32003c30dc19668f          • android-arm64  • Android 10 (API 29)
    • Taha’s iPhone (mobile) • 00008020-001059882212002E • ios            • iOS 13.6
    • macOS (desktop)        • macos                     • darwin-x64     • Mac OS X 10.15.6 19G73
    • Web Server (web)       • web-server                • web-javascript • Flutter Tools
    • Chrome (web)           • chrome                    • web-javascript • Google Chrome 84.0.4147.89

• No issues found!

```

</details> 

Closing as not reproducible issue. If you disagree, please write in the comments and I will reopen it.
Thank you
